qa/rpc-tests/wallet.py

```diff
@@ -293,6 +293,8 @@ def run_test (self):
         mytxdetails = self.nodes[2].gettransaction(mytxid)
         myvjoinsplits = mytxdetails["vjoinsplit"]
         assert_equal(0, len(myvjoinsplits))
+        assert_equal("joinSplitPubKey" not in mytxdetails)
+        assert_equal("joinSplitSig" not in mytxdetails)
```

Comment:
Gah, sorry, I made a mistake in my earlier suggestion:
```suggestion
        assert("joinSplitPubKey" not in mytxdetails)
        assert("joinSplitSig" not in mytxdetails)
```